portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -1,18 +1,50 @@
 package com.google.sps.servlets;
 
 import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Random;
+
+import com.google.gson.Gson;
```

Comment:
Remive blank line above. Also order the imports in alphabetical order.
Sp com.google is the first import. So spaces between imports